features/steps/steps.py

```diff
@@ -200,6 +234,65 @@ def step_impl(context, constraint, num, entity):
 
     handle_errors(context, errors)
 
+@then('Each {entity} must nest {constraint} {num:d} instance(s) of {other_entity}')
```

Comment:
I know this is confusing and I also don't necessarily agree on the wording, but at the moment, we have the terminology in gherkin and IFC reversed.

Gherkin|IFC
---|---
Each {entity} must nest|inst.IsNestedBy 
Each {entity} must be nested only by |inst.Nests

Reply:
It's changed in https://github.com/buildingSMART/ifc-gherkin-rules/commit/df79f1f62d7459a92e7a59cbaa0ab27eb89a5508
The wording is perhaps a bit confusing now, but Gherkin and IFC are consistent in this commit.